### ISSUE TITLE
AttributeError: 'module' object has no attribute 'Wikipedia' / bad magic number

### ISSUE THREAD

**Issue by ivanhercaz** [martin-majlis/Wikipedia-API#2]
Hi @martin-majlis!
First of all, thank you for this API. It seems a powerful and useful API. But I am having a very strange problem... When I call the main module (`Wikipedia`) and I try to get a page, I get the next error:

> AttributeError: 'module' object has no attribute 'Wikipedia'

I import the module with the name you show in the README.md (`import wikipediaapi`). What I write to test your API is simple and it shouldn't result in that error:

<pre lang="python">

import wikipediaapi

wiki = wikipediaapi.Wikipedia('es')

page = wiki.page('Wikipedia')
</pre>

Regards,
Iván

**Comment by martin-majlis** [comment]
Hi @ivanhercaz , thanks for using this library. :)

What is the output of following code?

```
import wikipediaapi
print(wikipediaapi.__version__)
print(dir(wikipediaapi))
```

Version should be at least (0, 3, 0).
If you have some lower version, you may try to upgrade it: `pip3 install -U wikipedia-api`

Regards,
Martin

**Comment by ivanhercaz** [comment]
It is so strange @martin-majlis... If I use your code, the output is the same: `AttributeError: 'module' object has no attribute '__version__'`. Then I uninstall and install it again, and I confirm that I have installed the the version 0.3.4. One detail, I run the script with `python`, but if it is developed in Python 3, I must to run it with `python3`. I think that solve this little mistake could solve the problem... but it shows another error (ImportError):

> ImportError: bad magic number in 'wikipediaapi': b'\x03\xf3\r\n'

Regards,
Iván

**Comment by ivanhercaz** [comment]
Excuse me, @martin-majlis! Everything has been my own error...

The problem was that I run the script with `python` and not `python3`, it produce an error. When I hit upon that I didn't run it with the correct command because Wikipedia API is in Python 3, I tried to run with `python3`. But the output show me the bad magic number error (what I said in the last comment). Finally, I found the solution in the net: [What's the bad magic number error?](https://stackoverflow.com/questions/514371/whats-the-bad-magic-number-error?rq=1) After reading that I run the `find` command with the `delete` parameter in my home folder to clean al pyc files. Then, everything works fine!

One more time, excuse me for that naive issue! If I achieve what I want to do, I will notify you 

Regards,
Iván
